### ISSUE TITLE
Support Wasabi on Android and iOS

### ISSUE THREAD

**Issue by nopara73** [zkSNACKs/Meta#9]

**Comment by wasaw936** [comment]
Im not entirely sure an Android coinmixer is the best choice for privacy

The following fallbacks/pitfalls come to mind:

- What about tor? every user will require orbot isnt that so?
- If orbot is used, then you have to think of Battery life?
- Phones are not reliable, What if the phone loses connectivity during a coinjoin?

I believe that one should mix coins on PC and then send to wallet.

I do think an android app which would link and send notifications of Wasabi app on PC to the Wasabi app, to let user know the progress of coinjoin, maybe?  App similar to bisq.

Thoughts?

**Comment by nopara73** [comment]
@wasaw936 I agree, however this can be reviewed a few years from now after we establish it's possible at all.

https://github.com/zkSNACKs/Meta#vi-accessibility

> In theory, Wasabi could support smart phones (Android, iOS). In practice, these platforms and their tools are not mature enough just yet. The concept of network analysis resistant smartphone wallets are not yet proven. If we would try to port Wasabi's code today, the wallet would use too much storage space, battery and network. Another implementation issue comes from Wasabi's reliance on Tor, which adds another layer of friction. On the desktop, we were able to work around nearly all of the reliability issues of the anonymity network, but the current state of Tor on mobile platforms may pose additional challenges.  
However, technology is improving quickly, thus, timing has special importance in this matter.

**Comment by MaxHillebrand** [comment]
This is an promising project, though in early development - a cross platform smartphone application powered by Wasabi

https://github.com/chaincase-cash/chaincase/
